_tools/fpki-stat-report

```diff
@@ -0,0 +1,113 @@
+
+![FPKIMA Logo]
+
+## Purpose
+
+This report provides a technical and policy compliance status for each Federal Public Key Infrastructure (FPKI) Affiliate.
+
+## FPKI Affiliate Scorecard
+
+The operational status for each FPKI Affiliate is summarized in the FPKI Affiliate Status Summary below.  An overall operational status include both issues that affect technical interoperability and non-compliance with applicable Certificate Policy (CP). Scorecards are not used for any other purpose such as ranking or rating. Affiliates can:
+
+- Resolve technical issues by contacting the [FPKIMA Team](fpkipa-ma@listserv.gsa.gov). 
+- Resolve Certificate Policy issues by contacting the [Certificate Policy Working Group (CPWG)](fpkipa_cpwg@listserv.gsa.gov).  
+
+## FPKI Affiliate Scorecard Legend
+
+**_Significant_** – Technical and/or policy issues that WILL adversely affect interoperability. </br> 
+**_Moderate_** – Technical and/or policy issues that MAY or MAY NOT adversely affect interoperability. </br>
+**_No Impact_** – Technical and/or policy issues that WILL NOT adversely affect interoperability. </br> 
+**_No Issues_** – No technical or policy issues were found in the last thirty days. </br>
+
+## FPKI Affiliate Status Summary
+
+The following table summarizes the Scorecard status of each connected CA to Federal Common Policy or Federal Bridge.
+
+| Federal Agency or Affiliate CA | FPKIMA CA | Status |
+| ------------------------------ | --------- | ------ |
+| CertiPath Bridge SHA-2 | FBCA | No Issues |
+| Department of Defense SHA-2 | FBCA | No Issues |
+| DigiCert NFI | FBCA | No Issues |
+| Entrust Managed Services NFI | FBCA | No Issues |
+| Exostar NFI | FBCA | No Issues |
+| Government Printing Office | FBCA | No Issues |
+| GSA Access Certificate for Electronic Services (ACES) | FBCA | No Issues |
+| IdenTrust NFI | FBCA | No Issues |
+| WidePoint/ORC NFI | FBCA | No Issues |
+| SAFE BioPharma Bridge | FBCA | No Issues |
+| STRAC Bridge | FBCA | No Issues |
+| TSCP Bridge | FBCA | No Issues |
+| US Patent and Trademark Office (PTO) | FBCA | No Issues |
+| Verizon Business NFI | FBCA | No Issues |
+| CertiPath Bridge SHA-1 | SHA1FRCA | No Issues |
+| Department of Defense SHA-1 | SHA1FRCA | No Issues |
+| Department of State | FCPCA | No Issues |
+| DigiCert/Symantec/Verisign SSP | FCPCA | No Issues |
+| Entrust Managed Services SSP | FCPCA | No Issues |
+| WidePoint/ORC SSP | FCPCA | No Issues |
+| US Treasury SSP | FCPCA | No Issues |
+| Verizon Business SSP | FCPCA | No Issues |
+
+## FPKI Trust Infrastructure Details
+
+### FPKIMA Certificate Activity
+
+The following certificates were issued **BY** or **TO** the FPKI Trust Infrastructure in the last 30 days.
+| Affiliate | Subject CA | Issuing CA | Serial | Issued Date |
+| --------- | ---------- | ---------- | ------ | ------ |
+| No Activity |
+
+The following certificates have been removed from the FPKI Trust Infrastructure in the last 30 days.
```

Comment:
Is this supposed to be inclusive of all CAs? Or only the ones that Common or the Federal Bridge CA sign?

Reply:
Just MA CAs because we don't have a good method to identify all CAs yet. Future enhancement to include all CAs.